mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -1732,6 +1910,9 @@
             <LayoutButtons Face="BurrowUp" Type="AbilCmd" AbilCmd="BurrowInfestorUp,Execute" Row="2" Column="4"/>
             <LayoutButtons Face="BurrowUp" Type="AbilCmd" AbilCmd="BurrowUltraliskUp,Execute" Row="2" Column="4"/>
         </CardLayouts>
+        <Radius value="0.875"/>
+        <InnerRadius value="0.6875"/>
+        <MinimapRadius value="0.875"/>
```

Comment:
Ultralisk: Reduced size by 12.5%
- unit radius (unit collision) was reduced from 1.0 to 0.875 (-12.5%)
- minimap radius was reduced from 1.0 to 0.875 (- 12.5%)
- inner radius (building collision) was reduced from 0.75 to 0.6875 (-8.3%)

The inner radius was only reduced by 8.3% instead of 12.5%. The value is 0.6875.
The editor requires values that are multiple of 0.0625 here, so this was chosen as the closest desired value